libs/api-authentication/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './lib/current-organization'
```

Comment:
Change this to explicit exports, barrel exports like this can cause issues apparently

Reply:
Done